Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.cpp

```diff
@@ -538,6 +447,15 @@ namespace EMotionFX
         const size_t numColliders = colliders.size();
         size_t numAvailableColliderWidgets = m_colliderWidgets.size();
 
+        if (numColliders)
```

Comment:
nit: Please use explicit equality check instead of implicit null check (`numColliders == 0`) (or `colliders.empty()`)